cluster/node-pools/master-default/stack.yaml

```diff
@@ -39,12 +36,6 @@ Resources:
       - Key: zalando.org/pod-max-pids
         PropagateAtLaunch: true
         Value: "{{ .NodePool.ConfigItems.pod_max_pids }}"
-      - Key: application
-        Value: kubernetes
-        PropagateAtLaunch: true
```

Comment:
Is this true for tags set at the CF stack level?

Reply:
Here, it says the tagging implementation can vary by resource, and that this `PropagateAtLaunch` is available only for `AutoScalingGroup` resources. 🤦 
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-resource-tags.html

Reply:
For resources that support tagging, tags propagate from stack level to those resources created by the stack. So, I believe this will be `true` and EC2 instances should have the tag added to the stack.